src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        if (fromFileName== null || toFileName == null){
+            return;
+        }
```

Comment:
The null check for file names is good, but it might be better to throw an IllegalArgumentException instead of just returning, to inform the caller about the incorrect usage.